packages/twenty-front/src/modules/object-record/multiple-objects/types/MorphItem.ts

```diff
@@ -0,0 +1,4 @@
+export type MorphItem = {
```

Comment:
Interesting, we should discuss about this type with Charles for the coming refactor on morph many.